src/Sulu/Bundle/AdminBundle/Resources/js/components/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import progressBarStyles from './progressBar.scss';
+
+type Props = {
+    max: number,
+    type: 'progress' | 'success' | 'error' | 'warning',
```

Comment:
i know i proposed `type` in the last review, but i think `skin` would probably the better name. i am sorry 🙈